accounts/migrations/0001_initial.py

```diff
@@ -8,17 +8,45 @@
 class Migration(SchemaMigration):
 
     def forwards(self, orm):
+        # Adding model 'TwitterInfo'
```

Comment:
Can you just revert all your changes to 0001_initial.py for the sake of this pull request? I don't see why we need these changes. After all migration files are just to keep a record of how the db has been migrated over a time. No one should be running these files going forward.